modules/virt-creating-interface-on-nodes.adoc

```diff
@@ -42,12 +45,25 @@ spec:
 <1> Name of the policy.
 <2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
 <3> This example uses the `node-role.kubernetes.io/worker: ""` node selector to select all worker nodes in the cluster.
-<4> Optional: Human-readable description for the interface.
+<4> Optional: Specifies the maximum number of nmstate-enabled nodes that the policy configuration can be applied to concurrently. This parameter can be set to either a percentage value, for example, 10%, or an absolute value, such as 3.
```

Comment:
Could we also somehow mention that in case of the percentage value, the value needs to be a string?
like:
maxUnavailable: "10%"
And in case of an absolute value, the value is a number:
maxUnavailable: 3

Reply:
Made the change

Reply:
@rhrazdil Can you review again? Thanks!